Add Multi-Chain IBC Transfers Example (#54)
subquery/cosmos-subql-starter: Multi-Chain/osmosis-cosmos-bridge/README.md

# SubQuery - Multi-Chain. Example project for IBC transfers between Osmosis and Cosmos Hub

[SubQuery](https://subquery.network) is a fast, flexible, and reliable open-source data indexer that provides you with custom APIs for your web3 project across all of our supported networks. To learn about how to get started with SubQuery, [visit our docs](https://academy.subquery.network).

> **The SubQuery project is responsible for indexing all IBC transfer events occurring between Cosmos Hub and Osmosis**

Reviewer: ```suggestion
**This SubQuery project indexes all IBC transfer events between Cosmos Hub and Osmosis**
```
